<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findClosest</class>
 <widget class="QWidget" name="findClosest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Store ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Store Area</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Items Available</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Daily Customers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Store Sales</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Feature value...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_2">
     <item>
      <property name="text">
       <string>Store ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Store Area</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Items Available</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Daily Customers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Store Sales</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="predictButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
